repositories/src/test/scala/cromwell/pipeline/datastorage/dao/repository/utils/TestUserUtils.scala

```diff
@@ -9,8 +9,10 @@ import cromwell.pipeline.utils.StringUtils
 object TestUserUtils {
   val userPassword = "-Pa$$w0rd-"
 
+  private def randomUuidStr: String = UUID.randomUUID().toString
+  def getDummyUserId = UserId(randomUuidStr)
   def getDummyUser(
-    uuid: String = UUID.randomUUID().toString,
+    uuid: String = randomUuidStr,
```

Comment:
It looks like this parameter is unused now. No matter what, you are calling `getDummyUserId` on line 25, which ignores this `uuid` parameter.
If you want to do it right, you must change the signature of this method. Instead of `uuid: String` it must accept `userId: UserId` with the default value `getDummyUserId`. Then this `userId` should be used on line 25.

Reply:
fixed